src/vstest.console/CommandLine/Executor.cs

```diff
@@ -93,7 +93,16 @@ internal int Execute(params string[] args)
         {
             this.testPlatformEventSource.VsTestConsoleStart();
 
-            this.PrintSplashScreen();
+            // If User specifies --nologo via dotnet, donot print splat screen
+            if (args != null && args.Length !=0 && args.Contains("--nologo"))
```

Comment:
Should we support `/nologo`? Reference https://msdn.microsoft.com/en-us/library/ms164311.aspx

Reply:
we are not supporting this as first class argument yet, only if it comes from dotnet cli, & from dotnet cli it will always come as --nologo. I don't want to add additional checks for something we are not supporting